Gemfile

```diff
@@ -1,16 +1,26 @@
-source 'https://rubygems.org'
-ruby '2.4.2'
+# frozen_string_literal: true
 
-gem 'sinatra'
-gem 'puma'
-gem 'foreman'
-gem 'redis'
-gem 'msgpack'
-gem 'hiredis'
-gem 'feedtosis', git: 'https://github.com/alown/feedtosis'
-gem 'moneta'
-gem 'builder'
+source "https://rubygems.org"
+ruby "2.5.0"
```

Comment:
stuck at 2.5.0 to deal with feedtosis, i want to address that first before this is ready.